### PR TITLE
Platform based alerts

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description

Depends on https://github.com/cloud-bulldozer/e2e-benchmarking/pull/385,

Also removing references to the deprecated LOG_STREAMING variable.

### CODE REVIEW

**Review by dry923 — APPROVED** [review]
LGTM!

File: dags/openshift_nightlies/config/benchmarks/control-plane.json [hunk 1]
```diff
@@ -33,7 +32,6 @@
                 "BURST": "20",
                 "STEP_SIZE": "30s",
                 "LOG_LEVEL": "info",
-                "LOG_STREAMING": "true",
                 "CLEANUP_WHEN_FINISH": "true",
                 "CLEANUP": "true",
                 "MUST_GATHER_EACH_TASK": "true",
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply dags/openshift_nightlies/config/benchmarks/control-plane.json:37]: Right, that's on purpose, when not set we use the default value set by e2e-bencharking script, which is 30m. 1h only for cluster-density tests.

[the remark this answers is not in the document]
Comment (Harshith-umesh) [reply dags/openshift_nightlies/config/benchmarks/control-plane.json:37]: ack got it

**Review by Harshith-umesh — APPROVED** [review]
LGTM

**Pull request merged**